src/progressbar/progressbar.spec.ts

```diff
@@ -19,7 +19,7 @@ function getBarHeight(nativeEl): string {
 }
 
 function getBarValue(nativeEl): number {
-  return parseInt(getProgressbar(nativeEl).getAttribute('aria-valuenow'), 10);
+  return parseInt(getProgressbar(nativeEl).getAttribute('aria-valuenow') || '', 10);
```

Comment:
`parseInt(null, 10)` or `parseInt('', 10)` returns undefined. 

`parseInt(<any>getProgressbar...` seems to me a better option, since it doesn't result to extra code.

Reply:
@fbasso good catch, haven't thought about extra runtime code